e2e/tests/omnibus-sepolia.toml/BFP_Market_Liquidations.e2e.js

```diff
@@ -0,0 +1,665 @@
+    await provider.send('evm_revert', [snapshot]);
+  });
+
+  it('should sync time of the fork', async () => {
+    await syncTime();
+  });
+
+  it('should create new random wallet', async () => {
+    log({ wallet: wallet.address, pk: wallet.privateKey });
+    assert.ok(wallet.address);
+  });
+
+  it('should set ETH balance to 100', async () => {
+    assert.equal(await getEthBalance({ address }), 0, 'New wallet has 0 ETH balance');
+    await setEthBalance({ address, balance: 100 });
+    assert.equal(await getEthBalance({ address }), 100);
+  });
+
+  it('should create user account', async () => {
+    assert.equal(
+      await getAccountOwner({ accountId }),
+      ethers.constants.AddressZero,
+      'New wallet should not have an account yet'
+    );
+    await createAccount({ wallet, accountId });
+    assert.equal(await getAccountOwner({ accountId }), address);
+  });
+
+  it('should set fWETH balance to 1_000', async () => {
+    assert.equal(
+      await getCollateralBalance({ address, symbol: 'fWETH' }),
+      0,
+      'New wallet has 0 fWETH balance'
+    );
+    const { tokenAddress } = await getCollateralConfig('fWETH');
+    await setMintableTokenBalance({ privateKey, tokenAddress, balance: 1_000 });
+    assert.equal(await getCollateralBalance({ address, symbol: 'fWETH' }), 1_000);
+  });
+
+  it('should approve fWETH spending for CoreProxy', async () => {
+    assert.equal(
+      await isCollateralApproved({
+        address,
+        symbol: 'fWETH',
+        spenderAddress: require('../../deployments/CoreProxy.json').address,
+      }),
+      false,
+      'New wallet has not allowed CoreProxy fWETH spending'
+    );
+    await approveCollateral({
+      privateKey,
+      symbol: 'fWETH',
+      spenderAddress: require('../../deployments/CoreProxy.json').address,
+    });
+    assert.equal(
+      await isCollateralApproved({
+        address,
+        symbol: 'fWETH',
+        spenderAddress: require('../../deployments/CoreProxy.json').address,
+      }),
+      true
+    );
+  });
+
+  it('should deposit 500 fWETH into the system', async () => {
+    assert.equal(await getCollateralBalance({ address, symbol: 'fWETH' }), 1_000);
+    assert.deepEqual(await getAccountCollateral({ accountId, symbol: 'fWETH' }), {
+      totalDeposited: 0,
+      totalAssigned: 0,
+      totalLocked: 0,
+    });
+
+    await depositCollateral({ privateKey, symbol: 'fWETH', accountId, amount: 500 });
+
+    assert.equal(await getCollateralBalance({ address, symbol: 'fWETH' }), 500);
+    assert.deepEqual(await getAccountCollateral({ accountId, symbol: 'fWETH' }), {
+      totalDeposited: 500,
+      totalAssigned: 0,
+      totalLocked: 0,
+    });
+  });
+
+  it('should delegate 500 fWETH into the Spartan Council pool', async () => {
+    assert.deepEqual(await getAccountCollateral({ accountId, symbol: 'fWETH' }), {
+      totalDeposited: 500,
+      totalAssigned: 0,
+      totalLocked: 0,
+    });
+    await delegateCollateral({
```

Comment:
there is no need to deposit/delegate/etc to be able to trade on perps. 
Is there any particular reason we have LP functions here in the test for Perps liquidations?

My only guess is that pool is literally empty and there is no way to trade... Just guessing